src/__main__.py

```diff
@@ -9,5 +10,7 @@
     discord.utils.setup_logging()
     logging.info("Starting ebot...")
 
-    bot = EBot()
+    admins = json.loads(os.environ["ADMIN_USERS"])
```

Comment:
Uhhh does that mean you have to do something like
```
ADMIN_USERS='["name1", "name2"]' python start-ebot
```
??

Reply:
With `pipenv` you can have a `.env` file in the root directory (which is ignored by the `.gitignore`), and so when you `pipenv shell` it'll load the env vars from that file. So it'd look like:
```
DISCORD_TOKEN=tokenGoesHere
ADMIN_USERS=[12345,67891]
```

Reply:
Ah nice okay